athena-federation-sdk/src/test/resources/serde/v2/ListTablesRequestPage.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
we dont have an object called ListTableRequestPage, so what is this?

Reply:
I wanted to have 2 different JSONs one where the nextToken is `null` and one where it actually contains a String value. This way I could test both scenarios (one where we paginate and one where we don't). But, maybe it's a distinction with no difference as far as the JSON parser is concerned.

Reply:
Removed